merqube_client_lib/secapi/client.py

```diff
@@ -80,7 +80,6 @@ def _validate_multiple(
         """Validate the input for functions that query for one or more securities"""
         self._validate_secapi_type(sec_type=sec_type)
 
-        assert sec_ids or sec_names, "Must provide either sec_ids or sec_names"
```

Comment:
This is not currently true; omitting both returns all securities. Whether it should.. different question.